demos/svg/lv_demo_svg.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * @file lv_demo_svg.c
```

Comment:
Please add it as an example instead.

Reply:
https://github.com/lvgl/lvgl/pull/6845/files#diff-c4ee2b6a8bbab84f9d61b9247509f95fae1d626aeb25a40681104e37e08f3498